libs/beacon/tests/unit/trusted_dealer_test.cpp

```diff
@@ -186,7 +186,6 @@ void RunTrustedDealer(uint16_t total_renewals = 4, uint32_t cabinet_size = 4,
       // Note, to avoid limiting the 'look ahead' entropy gen, set the block to ahead of numbers per
       // aeon
       member->beacon_service.MostRecentSeen((i * aeon_period) + aeon_period - 1);
-      member->setup_service.Abort(aeon_period);
     }
```

Comment:
Not needed as use of trusted beacon setup service means that the state where abort_below_ is check is never entered.